include/vinecopulib/misc/tools_stats.hpp

```diff
@@ -52,6 +54,52 @@ qnorm(const Eigen::MatrixXd& x)
   return tools_eigen::unaryExpr_or_nan(x, f);
 }
 
+//! @brief Quantile function of the Standard normal distribution with additional
+//! bound checks for numerical stability.
+//!
+//! @param x Evaluation points.
+//!
+//! @return An \f$ n \times d \f$ matrix of evaluated quantiles.
+inline Eigen::MatrixXd
+safe_qnorm(const Eigen::MatrixXd& x)
```

Comment:
* The important part is only =0 or =1 right? Because why would one evaluate quantiles at probabilities outside [0,1]?
* Any reason you don't rely on Eigen's ndtri?

Reply:
see below